### PR TITLE
feat(app): remove liquid setup FF

### PR DESCRIPTION
**Pull request by jerader**
closes RQA-260

# Overview

Removes the liquid setup FF from the app since liquid setup will be included in the 6.3 release

# Changelog

- remove liquid setup ff usage in all components
- deletes deprecated components no longer in use now (`HeaterShakerBanner`, `ModulesExtraAttention`)

# Review requests

- smoke test in the app, check out module, labware, and liquid setup by creating a protocol in PD with liquids in it. Check out protocol details, thumbnail, running a protocol (Note: there is a bug and Liquids key will be an empty array (https://opentrons.atlassian.net/browse/RLIQ-273) This affects the liquids tab from showing up in protocol details and the deck map rendering liquid in the wells)

# Risk assessment

low

### CODE REVIEW

**Review by koji — APPROVED** [review]
The changes look good to me.

I just noticed one thing. When open `How labware offsets work` modal.
Not sure why react forces us to use the lowercase for the prop name. 
Screenshot 2022-12-08 at 1 39 18 PM

**Comment by koji** [comment]
@jerader 
Seems that ModalShell has issues on props.

File: app/src/organisms/Devices/ProtocolRun/ProtocolRunSetup.tsx [hunk 1]
```diff
@@ -77,7 +75,6 @@ export function ProtocolRunSetup({
     let nextStepKeysInOrder = stepsKeysInOrder
     const showModuleSetup = protocolData != null && modules.length > 0
     const showLiquidSetup =
-      liquidSetupEnabled &&
       protocolData != null &&
       'liquids' in protocolData &&
       size(protocolData.liquids) > 0
```
Comment (smb2268) [inline app/src/organisms/Devices/ProtocolRun/ProtocolRunSetup.tsx]: we can change this to `protocolData.liquids.length > 0` since this will be an array

**Review by smb2268 — APPROVED** [review]
Looks good!! I am soo excited to remove more FF's

**Comment by jerader** [comment]
> I just noticed one thing. When open `How labware offsets work` modal. Not sure why react forces us to use the lowercase for the prop name. Screenshot 2022-12-08 at 1 39 18 PM

@koji, I will make a ticket to look into this. I am sure the error shows up for several modals

**Comment by koji** [comment]
> > I just noticed one thing. When open `How labware offsets work` modal. Not sure why react forces us to use the lowercase for the prop name. Screenshot 2022-12-08 at 1 39 18 PM
> 
> @koji, I will make a ticket to look into this. I am sure the error shows up for several modals

@jerader, thanks!

**Pull request merged**